File: _submissions/18UEE071_Ayush/Day4/amazingSubArray.py
class Solution:
    # @param A : string
    # @return an integer
    def solve(self, sd):
        container={'a':0,'e':0,'i':0,'o':0,'u':0,'A':0,'E':0,'I':0,"O":0,'U':0}
        s=0
        for i in range(len(sd)):
            ch = sd[i]
            if ch in container.keys():
                s+=(len(sd)-i)
        
        return(s%10003)

File: _submissions/18UEI056_Prashun/Day6/Amazing_subarray.cpp
bool isvowel(char x)
{ x=toupper(x);
    if(x=='A'||x=='E'||x=='I'||x=='O'||x=='U')
    return true;
    else
    return false;
}

int Solution::solve(string A) {
    if(A.size()==0) return 0;
    int len=A.length();
    int ans=0;
    for(int i=0;i<len;i++)
{  if(isvowel(A[i]))
     ans=ans+(len-i);
}
    return ans%10003;
}
